### PR TITLE
WIP - Various changes implimented from JaMmusic

### PR DESCRIPTION
**Pull request by Unesdala**
Made the changes from JaMmusic. The only differences now (besides changing some styling to make sure there wasn't glaring glitches, like with the size of the side logo) is the fact that the scss file is solely one file.

Going to take some time later and try to recreate it as close to 1:1 as I can with JaMmusic, so they're both fairly inline with each other. Need sleep before I do or I'd do it now.

### CODE REVIEW

**Comment by Unesdala** [comment]
May also need to add reactResizeDetector to other pages. Can test that out when I do the other thing.

**Comment by Unesdala** [comment]
@JoshuaVSherman Should be good to go now ^_^ Updated the scss files to be a 1:1 of JaM (Shy of removing the musicplayer and /music stuff that's specific to JaM, since it's unnecessary, and the few additional things that are in CollegeLutheran.)

Had to change the anchoring point for mobile (for the back to top feature), because it was scrolling a few pixels below.

File: static/_unsure.scss [hunk 1]
```diff
@@ -0,0 +1,134 @@
+/* I'm not honestly sure if this is still being used, but I don't want to delete it, just in case. Feel free to remove this otherwise. */
```
Comment (JoshuaVSherman) [inline static/_unsure.scss]: @Unesdala lets just delete this file please

**Comment by JoshuaVSherman** [comment]
@Unesdala also, the menu toggle button, please make the background color to be a square not rectangular:
image

**Comment by JoshuaVSherman** [comment]
@Unesdala the news and forum page is messed up again, please remove the border from the iframe and fix the column widths in the monthly forum table again

**Comment by Unesdala** [comment]
@JoshuaVSherman - This update should also cover a fix for [Scroll to Top on Mobile](https://github.com/WebJamApps/CollegeLutheran/issues/119#issue-576881908)

**Pull request merged**